src/components/OcImage.vue

```diff
@@ -36,6 +36,14 @@ export default {
       required: false,
       default: null,
     },
+    loadingType: {
```

Comment:
Loading type is not applied to the `img` tag in the template

Reply:
Might've lost this one during the rebases, brining it back now 💪🏽

Reply:
❤️